src/graph/executor/algo/IsomorExecutor.cpp

```diff
@@ -1,51 +1,46 @@
 // Copyright (c) 2020 vesoft inc. All rights reserved.
 //
 // This source code is licensed under Apache 2.0 License.
-
 #include "graph/executor/algo/IsomorExecutor.h"
-
 #include "graph/planner/plan/Algo.h"
-
 namespace nebula {
 namespace graph {
-
 folly::Future<Status> IsomorExecutor::execute() {
    // TODO: Replace the following codes with subgraph matching. Return type.
    SCOPED_TIMER(&execTime_);
    auto* isomor = asNode<Isomor>(node());
    DataSet ds;
    ds.colNames = isomor->colNames();
-
    auto iterDV = ectx_->getResult(isomor->getdScanVOut()).iter();
    auto iterQV = ectx_->getResult(isomor->getqScanVOut()).iter();
    auto iterDE = ectx_->getResult(isomor->getdScanEOut()).iter();
    auto iterQE = ectx_->getResult(isomor->getqScanEOut()).iter();
-
    unsigned int v_count = iterDV->size();
    unsigned int l_count = iterDV->size();
    unsigned int e_count = iterDE->size();
-
    unsigned int *offset = new unsigned int[v_count + 2];
    unsigned int *neighbors = new unsigned int[e_count * 2];
    unsigned int *labels = new unsigned int[l_count];
    // load data vertices id and tags
    while (iterDV->valid()) {
-     const auto v = iterDV->getColumn(nebula::kVid);  // check if v is a vertex
-     auto v_id = v.getInt();
-     const auto v2 = iterDV->getColumn(1);  // get label
-     auto l_id = v2.getInt();
+     const auto vertex = iterDV->getColumn(nebula::kVid);  // check if v is a vertex
+     auto v_id = vertex.getInt();
+     const auto label = iterDV->getColumn(1);  // get label by index
+     auto l_id = label.getInt();
      // unsigned int v_id = (unsigned int)v.getInt(0);
      labels[v_id] = l_id;  // Tag Id
```

Comment:
Can vID used as the index of the `labels` array? There is no restriction on what vID will look like, e.g., may be from 1000 to 100000.

Reply:
Therefore, it's better to use map here.